src/NHibernate.Test/Events/Collections/CollectionListeners.cs

```diff
@@ -158,6 +160,15 @@ public override void OnInitializeCollection(InitializeCollectionEvent @event)
 				base.OnInitializeCollection(@event);
 				AddEvent(@event, this);
 			}
+
+			public override Task OnInitializeCollectionAsync(InitializeCollectionEvent @event, CancellationToken cancellationToken)
```

Comment:
Async generator does not currently generates async counterparts for overrides of synchronous methods. See [#46](https://github.com/maca88/AsyncGenerator/issues/46). Some other test classes are affected and fixed too.